src/cljs/nr/routes.cljs

```diff
@@ -40,6 +40,8 @@
                       :view deck-builder}]
      ["/play" {:name :nav/lobby
                :view lobby-or-game}]
+     ["/replay/:rid" {:name :nav/replay-lobby
```

Comment:
Is this being used somewhere?

Reply:
We redirect to a `/replay/...` path when launching a replay and without this it was confusing the router. I don't use `rid` since we actually take it from an element in the page, but wanted to keep it for path matching purposes.